### DISCUSSION TITLE
Duplicate Deleted Options

### DISCUSSION THREAD

**Discussion by mirrorclone** [m4mallu/clonebot#12] [General]
Hello there.  Can you add a feature to delete the same files directly without bot indexing?  For example, if you find the same file, delete it, then there will be no continuous workload.

For example, if the same file is found while scanning the target channel, it should delete duplicate it directly.
🔥🔥🔥🔥

**Comment by m4mallu** [comment]
There are two types of users. Some users wish to delete the same immediately when found in the scan, others are not interested to delete the duplicate files, just want to create an index of the target only. In both of these cases, there is only one solution, Index them first, then prompt for deletion. That's why the bot is designed as. 

There is another bot working as in real-time deleting of duplicate files can be found [**HERE**](https://github.com/m4mallu/uniquify)
